.pre-commit-config.yaml

```diff
@@ -65,7 +65,7 @@ repos:
           - --comment-style
           - "|#|"
   - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.0.287
+    rev: v0.0.290
```

Comment:
291 came out 5 days ago.

Reply:
Ah, interesting! I've just kept it the same as the python dependency.